java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1016,23 +987,14 @@ public int cloneAsync(User loggedInUser, String channelLabel,
     private Object[] clone(User loggedInUser, String channelLabel,
```

Comment:
would it make sense to add unit tests for this method? (maybe in another PR ?)

Reply:
This one is tested indirectly (it's called by other methods (`clone`, `cloneAsOriginal`), which are tested by `testCloneErrata`.  I also added the `testCloneVendorErrata`, which should cover the cloning of vendor errata.